README.md

```diff
@@ -1,11 +1,12 @@
-[![banner](examples/tutorials/quick-start/media/PF_banner.png)](https://microsoft.github.io/promptflow/index.html)
+[![banner](examples/tutorials/quick-start/media/banner.png)](https://microsoft.github.io/promptflow/index.html)
 
 ------
 
 # Prompt flow
 
 [![Python package](https://img.shields.io/pypi/v/promptflow)](https://pypi.org/project/promptflow/)
 [![CLI](https://img.shields.io/badge/CLI-reference-blue)](https://microsoft.github.io/promptflow/reference/pf-command-reference.html)
+[![vsc extension](https://img.shields.io/visual-studio-marketplace/i/prompt-flow.prompt-flow?logo=Visual%20Studio&label=Extension%20install)](https://microsoft.github.io/promptflow/reference/pf-command-reference.html)
 [![Doc](https://img.shields.io/badge/Doc-online-green)](https://microsoft.github.io/promptflow/index.html)
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/promptflow)](https://pypi.org/project/promptflow/)
```

Comment:
consider split as two line with similar amount of items: 
![image](https://github.com/microsoft/promptflow/assets/47586720/9d0788e4-f516-4660-8411-a093b535478e)

Reply:
updated